### PR TITLE
Adds button and switch sounds when using machinery

### PR DESCRIPTION
**Pull request by alsoandanswer**
for example, clicking 'vend' on a vending machine plays a button sound.

Ported from Bay.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/machinery/machinery.dm [hunk 1]
```diff
@@ -231,7 +231,7 @@ Class Procs:
 
 /obj/machinery/CouldUseTopic(var/mob/user)
 	..()
-	if(istype (user, /mob/living/carbon))
+	if(clicksound && istype(user, /mob/living/carbon))
```
Comment (NonQueueingMatt) [inline code/game/machinery/machinery.dm]:
```suggestion
	if(clicksound && iscarbon(user))
```

File: code/game/objects/items/devices/radio/radio.dm [hunk 2]
```diff
@@ -756,3 +756,8 @@ var/global/list/default_medbay_channels = list(
 /obj/item/device/radio/phone/medbay/Initialize()
 	. = ..()
 	internal_channels = default_medbay_channels.Copy()
+
+/obj/item/device/radio/CouldUseTopic(var/mob/user)
+	..()
+	if(istype(user, /mob/living/carbon))
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/radio/radio.dm]:
```suggestion
	if(iscarbon(user))
```

File: code/game/machinery/machinery.dm (hunk 1 above)
Comment (mikomyazaki) [inline code/game/machinery/machinery.dm]:
```suggestion
	if(clicksound && iscarbon(user))
```

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -37,6 +37,8 @@
 	layer = 2.9
 	anchored = 1
 	density = 1
+	clicksound = "button"
+	clickvol = 40
```
Comment (mikomyazaki) [inline code/game/machinery/vending.dm]:
```suggestion

```
`clickvol` is 40 by default

File: code/game/objects/items/devices/radio/radio.dm (hunk 2 above)
Comment (mikomyazaki) [inline code/game/objects/items/devices/radio/radio.dm]:
```suggestion
	if(iscarbon(user))
```

**Pull request merged**